packages/calling-component-bindings/src/participantListSelector.ts

```diff
@@ -31,14 +31,17 @@ const convertRemoteParticipantsToParticipantListParticipants = (
 
         return {
           userId: toFlatCommunicationIdentifier(participant.identifier),
-          displayName: participant.displayName,
+          displayName:
```

Comment:
displayName is used in lots of selectors, is it only the participant list we will want this? Otherwise I would suggest doing this in the baseSelector's getDisplayName

Reply:
will do!